molo/core/models.py

```diff
@@ -331,6 +331,42 @@ def move(self, target, pos=None):
             for p in self.translations.all():
                 p.translated_page.move(target, pos='last-child')
 
+    def copy_languages(self, current_site, destination_site):
+        for language in self.languages.all():
+            if not destination_site.languages.languages.filter(
+                    locale=language.language.locale).exists():
+                new_lang = SiteLanguageRelation.objects.create(
+                    language_setting=Languages.for_site(destination_site),
+                    locale=language.language.locale,
+                    is_active=False)
+            else:
+                new_lang = destination_site.languages.languages.filter(
+                    locale=language.language.locale).first()
+            return new_lang
+
+    def copy(self, *args, **kwargs):
+        current_site = self.get_site()
+        destination_site = kwargs['to'].get_site()
+        if not (current_site is destination_site):
```

Comment:
This reads better for me `if current_site is not destination_site:` (without the brackets)